sample/kotlin/src/glide/kotlin/com/datadog/android/sample/picture/PictureViewModel.kt

```diff
@@ -29,5 +31,9 @@ class PictureViewModel : ViewModel() {
 
     companion object {
         const val RANDOM_URL = "https://source.unsplash.com/random/800x450"
+
+        fun setup(context: Context, okHttpClient: OkHttpClient) {
```

Comment:
why not providing this from the ViewModelFactory as we do for the `TracesViewModel` ?

Reply:
Because I need to setup Picasso only once per app session, otherwise it crashes

Reply:
Damn 😞